templates/artist/artist_detail.html

```diff
@@ -20,15 +20,15 @@
 {% block content %}
     <h1 class="header-font">{{ artist.name }}</h1>
     <ul class="tabs" data-tabs id="artist-detail-tabs">
-        <li class="tabs-title {% if not form.errors %}is-active{% endif %}"><a href="#overview" aria-selected="true">Overview</a></li>
+        <li class="tabs-title tab-formatting {% if not form.errors %}is-active{% endif %}"><a href="#overview" aria-selected="true">Overview</a></li>
```

Comment:
Rather than adding a new CSS class here, why not just use the existing `tabs-title`?